jfix-stdlib-socket/build.gradle.kts

```diff
@@ -1,13 +1,10 @@
-import org.gradle.kotlin.dsl.*
-
-
 plugins {
     java
     kotlin("jvm")
 }
 
 dependencies {
-    compile(Libs.slf4j_api)
+    api(Libs.slf4j_api)
```

Comment:
is it really an `api` depenedency? As I can see logger is a **private** logic of SocketChecker and ProxySocket

Reply:
I thought the client needed slf4j_api in his classpath to substitute his implementation of this API. It was my mistake - private logic is private logic.

To make sure it's works, I checked it on local environment using maven-local repo